src/index.js

```diff
@@ -2,7 +2,7 @@ import apiCreate from './api/create';
 import apiEmit from './api/emit';
 import apiFragment from './api/fragment';
 import apiInit from './api/init';
-import apiProperties from './api/properties/index';
```

Comment:
Yeah I did this and then realised that it was because I was including stuff wrong. Fixed.